redis_db.py

```diff
@@ -38,12 +35,12 @@ def process(self, nparray):
                 'decoder',
                 inputs=['d_input', 'hidden', 'e_output'],
                 outputs=['d_output', 'hidden'])
-            d_output = self.con.tensorget('d_output', as_type=rai.BlobTensor).to_numpy()
+            d_output = self.con.tensorget('d_output', as_numpy=True)
```

Comment:
`as_numpy=True` is the default behavior, so it might be better not to use it there specifically?

```suggestion
            d_output = self.con.tensorget('d_output')
```